meteor-app/imports/ui/pages/workspace/component.js

```diff
@@ -88,35 +100,31 @@ export default class WorkspacePage extends React.Component {
     toggleLayer(layerIndex, layerVisible);
   }
 
-  _layerOpacityOnChange (event) {
-    const target = event.currentTarget;
-    const layerIndex = parseInt(target.getAttribute('data-layer-index'), 10);
-    const opacity = target.value / 255;
+  _layerOpacityOnChange (context, value) {
+    const opacity = value / 255;
     const {
       updateLayerOpacity,
     } = this.props;
 
-    updateLayerOpacity(layerIndex, opacity);
+    updateLayerOpacity(context['data-layer-index'], opacity);
```

Comment:
`context['data-layer-index']` is not a reliable way of reading attributes of an element.